test/fixtures/ignore-list.css

```diff
@@ -0,0 +1,36 @@
+.selector {
+  background: red;
+}
+
+.anotherSelector {
+  background: orange;
+}
+
+#id .selector {
+  background: yellow;
+}
+
+p.selector {
+  background: green;
+}
+
+.selectorModifier {
+  background: cyan;
+}
+
+.selector.anotherSelector {
+  background: blue;
+}
+
+.selector .anotherSelector {
+  background: purple;
+}
+
+.anotherSelector.selector {
+  background: black;
+}
+
+.anotherSelector .selector{
+  background: white;
+}
+
```

Comment:
Let's change the selector to ignore, `.ignoreselector` so it's easier to understand

We also need to add a test for `.ignoreselector-notme`  (doesn't pass AFAICT)